frontend/public/components/nav.jsx

```diff
@@ -424,7 +415,8 @@ export const Navigation = ({ isNavOpen, onNavSelect }) => {
         <NavSection title="Compute" required={FLAGS.CAN_LIST_NODE}>
           <ResourceClusterLink resource="nodes" name="Nodes" />
           <ResourceNSLink resource={referenceForModel(MachineModel)} name="Machines" required={FLAGS.CLUSTER_API} />
-          <ResourceNSLink resource={referenceForModel(MachineSetModel)} name="Machine Sets" required={FLAGS.CLUSTER_API} isSeparated />
+          <ResourceNSLink resource={referenceForModel(MachineSetModel)} name="Machine Sets" required={FLAGS.CLUSTER_API} />
+          <NavItemSeparator />
```

Comment:
In native k8s, this will add a separate after Nodes with nothing below it. Better to check `FLAGS.MACHINE_CONFIG`